### ISSUE TITLE
Add documentation about deprecation

### ISSUE THREAD

**Issue by dmsimard** [ansible-collections/community.azure#25]
samccann mentioned that we can add a small bit of documentation about the deprecation so the collection docs doesn't end up with no content at all like community.kubernetes:
![Screenshot from 2021-11-03 11-07-46](https://user-images.githubusercontent.com/1291204/140087066-2a8986a9-2a6a-4188-85a9-0db70a30e09e.png)

We should be able to do that by adding a docsite directory a bit like in this PR: https://github.com/ansible-collections/amazon.aws/pull/362/files

**Comment by gundalow** [comment]
Good idea, should also update `README.md` to point to the new Repo and new collection on Galaxy (and delete the rest of the README.md)
